jetson/temp_camera/camera.py

```diff
@@ -0,0 +1,16 @@
+import jetson.utils
```

Comment:
Ok I've never seen this file before.... I don't think this belongs here? It originated from nkr/nucleo_bridge apparently.

Reply:
interesting that nucleo bridge has camera code in it..

Reply:
Uhhhh I guess its possible my members pushed code to the wrong branch?

Reply:
lol ok, if they have to get it back it's all in the commit history